src/content/docs/tutorial-create-alerts/create-new-relic-alerts.mdx

```diff
@@ -35,13 +35,14 @@ An **incident** is an individual event that details a symptom of a problem, whil
 
 In New Relic, you most often manage your alerts from [**one.newrelic.com > Alerts & AI**](https://one.newrelic.com/alerts-ai). This section of the platform has an **Overview** page that gives you visual breakdowns of all your alerts, as well as an **Alert conditions (policies)** page which gives you a list of all your alerting policies as well as a place to create new ones.
 
-This doc series deals primarily with how to set up basic alerting to give you a solid foundation for getting started and learning exploring the aspects of alerting that are most relevant to your organization. For a full overview of all our alerting features, check out our [alerting docs](/docs/alerts-applied-intelligence/overview/).
-
 ## What's in this series? [#alerts-series]
 
+This doc series deals primarily with how to set up basic alerting to give you a solid foundation for creating alerts using the any of the three most common creation options. It also teaches you how to manage the quality of your alerts, use your created alerts to find information about your system, and improve your stack using the information your alerts have given you.
```

Comment:
`common creation options.` what does this mean?

Reply:
`use your created alerts to find` and `using the information your alerts have given you.` are both a bit complex for my taste.